### PR TITLE
rename skycoord_to_string to something more accurate and descriptive

### PR DESCRIPTION
**Pull request by eteq**
“skycoord_to_string” is poorly named because it’s not for converting SkyCoord’s *in general* into strings, but rather length-4 SkyCoords into the right kind of list for the query.  So I’d suggest this renaming.

### CODE REVIEW

**Comment by abostroem** [comment]
Thanks for the PR @eteq! Your point is well taken but I'm worried that the new variable name is too long. Would you be happy with either as a compromise?:
`coord_list_to_str`, `coord_list_to_string`, or `poly_coord_to_str`?

File: _episodes/02-coords.md [hunk 1]
```diff
@@ -596,8 +596,8 @@ This is something we will need to do multiple times. We will write a function to
 The following function combines these steps. 
 
 ~~~
-def skycoord_to_string(skycoord):
-    """Convert SkyCoord to string."""
+def polygon_skycoord_to_gaia_string(skycoord):
```
Comment (ErinBecker) [inline _episodes/02-coords.md:599]:
```suggestion
def skycoord_to_string(skycoord):
```

File: _episodes/02-coords.md [hunk 2]
```diff
@@ -607,7 +607,7 @@ def skycoord_to_string(skycoord):
 Here is how we use this function:
 
 ~~~
-sky_point_list = skycoord_to_string(corners_icrs)
+sky_point_list = polygon_skycoord_to_gaia_string(corners_icrs)
```
Comment (ErinBecker) [inline _episodes/02-coords.md:610]:
```suggestion
sky_point_list = skycoord_to_string(corners_icrs)
```

File: _episodes/02-coords.md (hunk 1 above)
Comment (ErinBecker) [reply _episodes/02-coords.md:599]: I recommend keeping the shorter function name, but expanding the function description.

**Comment by ErinBecker** [comment]
@abostroem - I've suggested keeping the shorter function name, but expanding the function description to be more explicit.

**Comment by abostroem** [comment]
Don't forget to update episode_functions.py

**Pull request merged**